### PR TITLE
Added 2 courses in Urdu

### PR DESCRIPTION
**Pull request by morhadi**
## What does this PR do?
Add resource(s) 
* created free-courses-ur.md
* added 2 courses

### Why is this valuable (or not)?
This is the first contribution in Urdu language

### How do we know it's really free?
they are freely available youtube playlists

### For book lists, is it a book? For course lists, is it a course? etc.
* youtube playlists

## Checklist: (done ✅)
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-ur.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<div dir="rtl" markdown="1">
+
+### Index
+
+* [C#&lrm;](#csharp)
```
Comment (eshellman) [inline courses/free-courses-ur.md:5]:
```suggestion
* [C#](#csharp)
```

File: courses/free-courses-ur.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+* [HTML and CSS](#html-and-css)
+
+
+### <a id="csharp"></a>C#&lrm;
+
+* [C# Tutorials In Urdu/Hindi] (https://youtube.com/playlist?list=PLUyYwyJA_WfQd5zeCU890TDFQAqboekyc&si=Yjfl_cr5Ssu22Z3b) - ProgramInUrdu
```
Comment (eshellman) [inline courses/free-courses-ur.md:11]:
```suggestion
* [C# Tutorials In Urdu/Hindi](https://youtube.com/playlist?list=PLUyYwyJA_WfQd5zeCU890TDFQAqboekyc) - ProgramInUrdu
```
remove tracking parameter

File: courses/free-courses-ur.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+
+### HTML and CSS
+
+* [HTML5 & CSS3 Tutorials In Urdu and Hindi] (https://youtube.com/playlist?list=PLUyYwyJA_WfTr3YWWJ41_V7TrRZoq6cBT&si=1ymTOgZ2s8D1rI4y) - ProgramInUrdu
```
Comment (eshellman) [inline courses/free-courses-ur.md:16]:
```suggestion
* [HTML5 & CSS3 Tutorials In Urdu and Hindi](https://youtube.com/playlist?list=PLUyYwyJA_WfTr3YWWJ41_V7TrRZoq6cBT) - ProgramInUrdu
```

File: courses/free-courses-ur.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<div dir="rtl" markdown="1">
```
[the remark this answers is not in the document]
Comment (morhadi) [reply courses/free-courses-ur.md]: Removed

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**